internal/wscontents/wscontents.go

```diff
@@ -427,6 +427,12 @@ func handleEvents(ctx context.Context, debugLogger, userVisible io.Writer, absPa
 			return nil, false, fnerrors.InternalError("rel failed: %v", err)
 		}
 
+		// Ignore changes to files which are not valid paths for example includes ..
```

Comment:
So if I understand correctly you see a path that is outside of `absPath`, which then leads to the resulting path being "../foobar" here? I.e. resolves to a relative path outside of absPath.

If that's the case, then I'd check for relative paths instead, e.g.

```
// Observed a change that is not within the destination we're observing. We'll ignore it.
if strings.HasPrefix(rel, "../") {
  fmt.Fprintf(..)
  continue
}
```

That would be slightly preferred vs fs.ValidPath, as ignoring other "invalid" paths may mask other bugs.

Reply:
Ok. I updated the pull request.